### PR TITLE
Update reportportal-client to version 5.5.0

### PR DESCRIPTION
**Pull request by AmsterGet**
* Update [reportportal-client](https://github.com/reportportal/client-javascript) to version 5.5.0.
* Add [rerun](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/rerun.md) support.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -49,16 +49,17 @@ class ReportPortalHelper extends Helper {
 
     return rpClient.startLaunch({
       name: this.config.launchName || suiteTitle,
-      start_time: rpClient.helpers.now(),
```
Comment (PeterNgTr) [inline index.js]: @AmsterGet don't we need the `startTime` anymore?

File: README.md [hunk 2]
```diff
@@ -25,21 +25,28 @@ Example:
...
--`endpoint`: your reportportal host + `api/v1` for instance: `http://localhost:8080/api/v1`
--`launchName`: the launch name you want, if not provided, the suite title will be used
--`launchDescription`: the description of your launch, if not provided, the description will be empty
--`projectName`: the project that you created in the reportportal UI
--`debug`: to turn on the debug for reportportal
+- `token`: which can be found by navigating to the user profile page, clicking the username drop-down in the right header and selecting the "Profile" option > "UUID" – is a unique user identifier. UUID is used in automated test configuration files for a user authentication instead of a password. It will allow you to post data, without logging it in the UI.
+- `endpoint`: your reportportal host + `api/v1` for instance: `http://localhost:8080/api/v1`
+- `launchName`: the launch name you want, if not provided, the suite title will be used
+- `launchDescription`: the description of your launch, if not provided, the description will be empty
+- `launchAttributes`: the description of your launch, if not provided, the description will be empty
```
Comment (PeterNgTr) [inline README.md]: this is launch attributes :)

File: index.js [hunk 3]
```diff
@@ -49,16 +49,17 @@ class ReportPortalHelper extends Helper {
...
       description: this.config.launchDescription || '',
+      attributes: this.config.launchAttributes || [],
+      rerun: this.config.rerun || false,
+      rerunOf: this.config.rerunOf || undefined,
```
Comment (PeterNgTr) [inline index.js]: could you give the references of those?

File: index.js (hunk 1 above)
Comment (AmsterGet) [reply index.js]: As you can see [here](https://github.com/reportportal/client-javascript/blob/master/lib/report-portal-client.js#L144) client add `startTime` automatically.

File: index.js (hunk 3 above)
Comment (AmsterGet) [reply index.js]: Yes, of course.
I've already added the link [here](https://github.com/PeterNgTr/codeceptjs-rphelper/pull/10/files#diff-04c6e90faac2675aa89e2176d2eec7d8R49).
https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/rerun.md#specified-launch

File: README.md (hunk 2 above)
Comment (AmsterGet) [reply README.md]: My mistake, sorry )

File: index.js (hunk 1 above)
Comment (PeterNgTr) [reply index.js]: cool! Thanks

**Comment by PeterNgTr** [comment]
https://app.codacy.com/manual/PeterNgTr/codeceptjs-rphelper/pullRequest?prid=5017442

Appreciation on your markdown file fixed :)

File: README.md [hunk 4]
```diff
@@ -25,21 +25,29 @@ Example:
...
+
+    - `token`: which can be found by navigating to the user profile page, clicking the username drop-down in the right header and selecting the "Profile" / "UUID" – is a unique user identifier. UUID is used in automated test configuration files for a user authentication instead of a password. It will allow you to post data, without logging it in the UI.
+    - `endpoint`: your reportportal host + `api/v1` for instance: `http://localhost:8080/api/v1`
+    - `launchName`: the launch name you want, if not provided, the suite title will be used
+    - `launchDescription`: the description of your launch, if not provided, the description will be empty
+    - `launchAttributes`: the attributes of your launch, if not provided, the attributes will be empty
+    - `projectName`: the project that you created in the reportportal UI
+    - `debug`: to turn on the debug for reportportal
+    - `rerun`: to enable rerun
+    - `rerunOf`: UUID of launch you want to rerun. If not specified, report portal will update the latest launch with the same name
```
Comment (PeterNgTr) [inline README.md]: @AmsterGet have you tried `rerun` without `rerunOf`? I got an error

```sh
(node:64016) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 10)
(node:64016) UnhandledPromiseRejectionWarning: Error: Request failed with status code 404: {"errorCode":4041,"message":"Launch 'This is awesome' not found. Did you use correct Launch ID?"}
    at /Users/thanhnguyen/Desktop/codeceptjs-rest-demo/node_modules/reportportal-client/lib/rest.js:27:23
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:64016) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 11)
```

Comment (AmsterGet) [reply README.md]: If the `rerunOf` option not specified, items from the launch will be added to the latest launch with the same name (https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/rerun.md#latest-launch).

Comment (PeterNgTr) [reply README.md]: I see, cause with my previous tests, I removed all launches =))

**Comment by PeterNgTr** [comment]
Thanks! All good!

**Pull request merged**